.travis.yml

```diff
@@ -26,7 +26,7 @@ install:
   - pip install -qq --upgrade pip setuptools wheel
   - pip install --upgrade numpy scipy | cat  # TF usually needs the latest Numpy
   - pip install -r requirements.txt | cat
-  - pip install $TF_PACKAGE | cat
+  - bash deps/install_tensorflow_1_15.sh
```

Comment:
It should install `$TF_PACKAGE` here, **not** TF 1.15.

Reply:
Also, because of that, the install Bash script is not so nice.
In any case, I would try to avoid using the Bash script here if possible. Maybe I will just enabled `-DNDEBUG` globally now, so that the patching should not be needed anymore. See also the discussion in #87.

Reply:
Btw, also check [Travis](https://travis-ci.org/rwth-i6/returnn/builds/621529097): It seems it works fine without this.

Reply:
OK will do.